packages/modular-scripts/src/esbuild-scripts/api.ts

```diff
@@ -153,6 +161,16 @@ function compileIndex({
     (node) => node.nodeName === 'body',
   ) as parse5.Element;
 
+  if (styleImports) {
+    [...styleImports].forEach((importUrl) =>
+      head.childNodes.push(
+        ...parse5.parseFragment(
+          `<link rel="stylesheet" href="${importUrl}"></script>`,
```

Comment:
Might this be simpler to read with `insertAdjacentHTML`

```typescript
styleImports?.length && head.insertAdjacentHtml('beforeEnd', 
  styleImports.map(importUrl => `<link rel="stylesheet" href="${importUrl}" />`).join('')
);
```

Note - there is a typo in the template being used - it opens a `link` and closes a `script` 

```suggestion
          `<link rel="stylesheet" href="${importUrl}" />`,
```

Reply:
> Might this be simpler to read with insertAdjacentHTML

`parse5.Element` instances do not seem to support `insertAdjacentHTML`. I think parse5's API is reduced compared to the real DOM API.

> Note - there is a typo in the template being used - it opens a link and closes a script

Lol, a copy paste went wrong. Thanks for spotting it.